content/5_Plat/Bitsets.problems.json

```diff
@@ -2,15 +2,8 @@
   "MODULE_ID": "bitsets",
   "school": [
     {
-      "uniqueId": "cses-1706",
-      "name": "School Excursion",
```

Comment:
not sure why you deleted this part of the json

Reply:
![image](https://github.com/cpinitiative/usaco-guide/assets/59292725/c849e170-20ed-4968-98ff-d2afc9ad0b81)
I did it because it says to delete all other fields. But I'm going to put them back

Reply:
It should be ok now

Reply:
> I did it because it says to delete all other fields.

updated the wording, hopefully it's clearer now